admin/tools/docker/build-docker-img.sh

```diff
@@ -65,37 +64,65 @@ if [ $# -lt 1 ] ; then
     exit 2
 fi
 
-if [ -z "$DEPDIR" -a -z "$QSERV_DIR" ] ; then
-    printf "ERROR: directory containing dependency scripts not specified" 
+if [ -z "$DEPS_DIR" -a -z "$QSERV_DIR" ] ; then
+    printf "ERROR: directory containing dependency scripts not specified"
     usage
     exit 3
 fi
-test "$DEPDIR" || DEPDIR=$QSERV_DIR/admin/bootstrap
+test "$DEPS_DIR" || DEPS_DIR=$QSERV_DIR/admin/bootstrap
 
 DOCKERDIR=$1
 shift
 
 # strip trailing slash
 DOCKERDIR=$(echo $DOCKERDIR | sed 's%\(.*[^/]\)/*%\1%')
-DEPDIR=$(echo $DEPDIR | sed 's%\(.*[^/]\)/*%\1%')
+DEPS_DIR=$(echo $DEPS_DIR | sed 's%\(.*[^/]\)/*%\1%')
```

Comment:
Same question here, why is it bad to have slash at the end, does it break anything?